contracts/contracts/eosio.system/src/exchange_state.cpp

```diff
@@ -6,76 +6,6 @@
 
 namespace eosiosystem {
 
-   asset exchange_state::convert_to_exchange( connector& reserve, const asset& payment )
```

Comment:
I propose to leave these methods here without actual use for now

Reply:
ok